packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -235,7 +251,8 @@ class ExpansionTile extends StatefulWidget {
   /// Used to override to the [ListTileThemeData.textColor].
   ///
   /// If this property is null then [ExpansionTileThemeData.textColor] is used. If that
-  /// is also null then the value of [ListTileThemeData.textColor] is used.
+  /// is also null then and [ThemeData.useMaterial3] is true, color of the [TextTheme.bodyLarge]
```

Comment:
Generally speaking, we do not rely on the TextStyle.color of text theme colors. Presumably there's a token that covers what the text color should be (here and elsewhere).

Reply:
Since there is no official expansion tile componenT. 
M2 `ExpansionTile`simply uses `ListTile.title` color and I did the same here for M3, `expasion_tile_template.dart` uses a list tile title token for this color.

cc: @guidezpl 
Is there way a better?